settings.js

```diff
@@ -68,7 +72,9 @@ function generate(){
             "icu_folding",
             "trim",
             "custom_name",
-            "street_suffix",
+            "street_synonyms_en",
+            "street_synonyms_usps",
```

Comment:
How about having something with a regex like `street_synonyms_*` and will fetch all the files ?
This may help for #393 or if we need to add custom street_synonyms by lang.

Reply:
Yeah good point, we should rethink these a bit and make them easier to customise.

Right now we have fairly limited synonyms coverage (ie. 'just the basics').
I'd like to explore what would happen if we added significantly more synonyms, I suspect that the impact would be very positive for search quality and hopefully doesn't affect performance much.

@Joxit could you please tell me a bit about how you're adding custom synonyms for your builds?

I'm guessing you're adding a bunch of French-specific and some other ones which you include in the `custom_street.txt` file?

Pending successful testing of this I would next like to expand on French, German and Spanish synonyms.

Reply:
This is a bit annoying but better since #375 and #407 

I have a fork [joxit/pelias-schema](https://github.com/Joxit/pelias-schema), when I start a new build, I fetch the upstream and add [this commit](https://github.com/Joxit/pelias-schema/commit/514fb027cb5dd3b2be5ad1f4da62db0f24f20951).

For the record, I added `cutom_street` in `peliasPhrase` and `peliasQuery` in `build-2019-09` (before `pelias/parser` release). `peliasQuery` is used for autocomplete where we can find streets names... And `peliasPhrase` because `street_suffix` is also present.

I add my synonyms in `synonyms/custom_street.txt` 
```
bd, bld, blvd, boul, bvd, boulevard #only bld is missing in this PR
ave, av, avenue # OK with this PR
saint, st, street # missing
sainte, ste, suite # missing
```
This list was created from our common uses (when we test the pelias). I have always been afraid of slowing down queries with too many synonyms :sweat_smile:.

Reply:
Do you possibly know of an official source which publishes a French equivalent of this?
https://pe.usps.com/text/pub28/28apc_002.htm

Reply:
I've opened https://github.com/pelias/schema/pull/449 to hopefully clean up your build process a little.

Ideally you shouldn't have to do any `synonyms` work in `peliasQuery`, I think you can remove that and you'll get a small perf benefit?

Reply:
I found this one : https://blog.bureaudeposte.net/rediger-une-adresse-aux-normes-postales/ for abbreviations

Name| abbreviation
-- | --
Allée | ALL
Avenue | AV
Boulevard | BD
Carrefour | CARR
Chaussée | CHS
Chemin | CHEM
Cours | CRS
Faubourg | FG
Immeuble | IMM
Impasse | IMP
Lieu-dit | LD
Lotissement | LOT
Montée | MTE
Passage | PAS
Place | PL
Résidence | RES
Route | RTE
Ruelle | RLE

Reply:
Next time I will check differences between synonyms in `peliasQuery` and #449 

Oh, and saint/sainte aren't for street suffix/prefix but a workaround for street names... :sweat_smile: